utils/releasetools/build-config.json

```diff
@@ -12,6 +12,12 @@
           "type": "deb",
           "platform": "focal"
         },
+        {
+          "arch": "x86_64",
+          "target": "ubuntu24.04",
```

Comment:
@roshkhatri I see we are skipping 22.04.  I'm Okay adding these, but maybe we should document our strategy here somewhere?

Reply:
Like focus on Ubuntu LTS releases would make sense to me.

Reply:
@roshkhatri @bbarani Can we make an issue to document our strategy and follow through on updating the website. I think I'll merge this now and we can follow up on the next release adding these images.

Reply:
Yes I have opened an issue for the website repo here https://github.com/valkey-io/valkey-io.github.io/issues/164.

Reply:
Also I think there is a problem, for compiling, we use https://github.com/marketplace/actions/run-on-architecture for the binaries for ARM architecture. Github ARM runners are in GA but as github-hosted runners, there are not in opensource according to https://github.com/orgs/community/discussions/19197#discussioncomment-10550689.

I  will look into more options.